src/pubsub/src/publisher/options.rs

```diff
@@ -44,18 +44,21 @@ impl BatchingOptions {
     /// Set the [BatchingOptions][Self::byte_threshold] field.
     ///
     /// # Example
-    /// ```
+    /// ```ignore
     /// # use google_cloud_pubsub::options::publisher::BatchingOptions;
     /// let options = BatchingOptions::new().set_byte_threshold(1000_u32);
     /// ```
-    pub fn set_byte_threshold<V: Into<u32>>(mut self, v: V) -> Self {
+    // TODO(#3686): support byte thresholds.
+    #[allow(dead_code)]
+    pub(crate) fn set_byte_threshold<V: Into<u32>>(mut self, v: V) -> Self {
         self.byte_threshold = v.into();
         self
     }
 
     /// Set the [BatchingOptions][Self::delay_threshold] field.
     ///
     /// # Example
+    ///
```

Comment:
We typically do not include a blank line here. Is it needed?

Reply:
Nope.